File: Submissions/Educational_041/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);

    int n = sc.nextInt();
    int m = sc.nextInt();
    
    int[] count = new int[n];
    for (int i = 0; i < m; i++) {
      int c = sc.nextInt();
      count[c - 1]++;
    }

    sc.close();

    int answer = m + 1;

    for (int i = 0; i < n; i++) {
      answer = Math.min(answer, count[i]);
    }
    
    System.out.println(answer);
  
  }

}

File: Submissions/Educational_041/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int n, m;
  cin >> n >> m;

  vector<int> count(n);
  
  for (int i = 0; i < m; i++) {
    int c;
    cin >> c;
    count[c - 1]++;
  }

  int answer = m + 1;
  for (int i = 0; i < n; i++) {
    answer = min(answer, count[i]);
  }

  cout << answer << endl;

}